training/src/main/java/wolox/training/models/Users.java

```diff
@@ -37,16 +37,29 @@
     @NotNull
     @ApiModelProperty(notes = "birthdate of user")
     private LocalDate birthdate;
+    @NotNull
+    @ApiModelProperty(notes = "password of user")
+    private String password;
```

Comment:
The password must be write-only access, you can use this annotation  @JsonProperty(access = Access.WRITE_ONLY)